src/__tests__/controller/comment.test.js

```diff
@@ -0,0 +1 @@
+test.only('some test', () => {});
```

Comment:
The folder name here should be `controllers` in the path `src/__tests__/controllers` and not `controller`.